specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-10-01/common-types.json

```diff
@@ -131,6 +131,32 @@
           "readOnly": true
         }
       }
+    },
+    "CostManagementProxyResource": {
```

Comment:
>CostManagementProxyResource [](http://example.com/codeflow?start=5&length=27)

Is this used anywhere? you can just use the proxy resource type from common types instead

Reply:
What was different, is it just the etag? Or is there more?

Reply:
Yes, just eTag.